consumer/src/main/java/com/flipkart/varadhi/consumer/ordering/SubscriptionGroupsState.java

```diff
@@ -0,0 +1,18 @@
+package com.flipkart.varadhi.consumer.ordering;
+
+import com.flipkart.varadhi.consumer.MessageTracker;
+import jakarta.annotation.Nullable;
+
+public interface SubscriptionGroupsState {
+    @Nullable
+    QueueGroupPointer getPointer(String groupId);
```

Comment:
GroupPointer ?

Reply:
ya :)
It is a pointer of a group in a Q.

I am still evaluating this name :).

This or I will revert to GroupState. Not finalized. The only Issue with GroupState is that it is very generic name.